packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/legacy/template.html

```diff
@@ -0,0 +1,23 @@
+<div class="datacenter-backup-legacy">
+    <oui-message type="warning">
+        <p>
+            <strong
+                data-translate="datacenter_backup_legacy_message_1"
```

Comment:
```suggestion
                data-translate="dedicatedCloud_datacenter_backup_legacy_message_1"
```

to be consistence, prefix all translations with "dedicatedCloud_"